src/ui/pages/Notifications/Notifications.tsx

```diff
@@ -0,0 +1,159 @@
+import { useState } from "react";
+import {
+  IonButton,
+  IonChip,
+  IonIcon,
+  IonList,
+  useIonViewWillEnter,
+} from "@ionic/react";
+import { settingsOutline } from "ionicons/icons";
+import { TabLayout } from "../../components/layout/TabLayout";
+import { useAppDispatch, useAppSelector } from "../../../store/hooks";
+import { setCurrentRoute } from "../../../store/reducers/stateCache";
+import { TabsRoutePath } from "../../../routes/paths";
+import "./Notifications.scss";
+import { i18n } from "../../../i18n";
+import { timeDifference } from "../../utils/formatters";
+import { getNotificationsCache } from "../../../store/reducers/notificationsCache";
+import { NotificationItem } from "./NotificationItem";
+import { Agent } from "../../../core/agent/agent";
```

Comment:
Remove unused import

Reply:
I am going to use that in my next PR:

>Clicking and interacting with the items (we have [another ticket](https://cardanofoundation.atlassian.net/browse/DTIS-942) for that)

It's a feature branch, won't be merged until everything is done. I left that so I remember.